packages/qdrant-loader-core/src/qdrant_loader_core/llm/providers/bedrock.py

```
@@ -0,0 +1,299 @@
+from datetime import UTC, datetime
+from typing import Any
+
+try:
+    import boto3
+    from botocore.exceptions import (
+        BotoCoreError,
+        ClientError,
+        EndpointConnectionError,
+        NoCredentialsError,
+    )
+except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)